src/index.ts

```diff
@@ -46,6 +46,11 @@ const parseArguments = (argv: string[]) =>
             type: "string",
             description: "Path or URL to custom schema"
           })
+          .option("non-interactive", {
+            alias: "ni",
```

Comment:
1. I dont think the alias is worth
2. I think the type should be a boolean (it is or not in interactive mode, there is no inbetween, dont forget to update the code for the check)